packages/code-studio/src/dashboard/linker/LinkerUtils.tsx

```diff
@@ -3,10 +3,37 @@ import { TableUtils } from '@deephaven/iris-grid';
 import Log from '@deephaven/log';
 import { ChartPanel, IrisGridPanel, DropdownFilterPanel } from '../panels';
 import LayoutUtils from '../../layout/LayoutUtils';
-import LinkType from './LinkType';
 
-const log = Log.module('LinkerLink');
+export type LinkType = 'invalid' | 'filterSource' | 'tableLink';
 
+export type LinkPointUntyped = {
+  panelId: string;
+  panelComponent?: string;
+  columnName: string;
+};
+
+export type LinkPoint = LinkPointUntyped & {
+  columnType: string;
+};
```

Comment:
Idk if changing these to `LinkPoint` and `TypedLinkPoint` would be better. Not sure about naming conventions really. Or `LinkPointWityType`.

I also only see `LinkPointUntyped` used in 1 place and I'm not sure why these couldn't just be a single type. If you wanted a partial type, you could use `Pick<LinkPoint, 'panelId' | 'columnName'>` in that spot instead if you only want to enforce it has those 2 props and they're always the same type as `LinkPoint` here.

Reply:
Yea I just got rid of LinkPointUntyped - I don't think it was actually needed.